src/pages/pricing/index.tsx

```diff
@@ -187,46 +207,38 @@ export default function Pricing() {
       <div className="container my-5">
         <Stack>
           <div className="container">
-            <div className="d-flex flex-row justify-content-center">
-              <div className="btn-group">
-                <button
-                  type="button"
-                  className={classnames("btn btn-outline-primary", {
-                    active: paymentModel === "course",
-                  })}
-                  onClick={() => updatePaymentModel("course")}
-                >
-                  <span
-                    className={classnames("me-2", {
-                      "d-none": paymentModel !== "course",
-                    })}
+            <Row>
+              <Col xs={12} md={6}>
+                <Form.Group controlId="term-system">
+                  <Form.Label>Term system</Form.Label>
```

Comment:
Is this the right term (heh) to use here?

Reply:
"Term system" is a correct terminology. We could also consider "academic schedule"

Reply:
I prefer "academic schedule", but I agree that "term system" is correct as well.

Reply:
Changed to "academic calendar" in 7129fb5.

Reply:
err, no I didn't 😅 let me change it again.

Reply:
Done for real in 56ec012.